.changelog/1831.txt

```diff
@@ -0,0 +1,5 @@
+```release-note:feature
```

Comment:
Just a general concern: I feel like people might assume that deleting a user from their OIDC provider will revoke their access to waypoint, but I don't think that will happen in our implementation.

Reply:
Will not this for the docs, I don't think any of our OIDC implementations enforce this right now. That's what the token expiration is about (and why GitHub makes you re-OIDC every single damn day lol).